### PR TITLE
docs(quarks): Sprint 12 update Quarks API Docs links

### PR DESCRIPTION
**Pull request by 100stacks**
### Sprint Docs 12 Quarks Maintenance 

- [x] Dev LGTM

---
Update `src/quarks/quarks.module.js` with remaining new quarks added during Sprint Docs 12.

### CODE REVIEW

**Comment by 100stacks** [comment]
Update `src/quarks/quarks.module.js` with the following PRs:

* TBD - `rxStatus.StatusUtil`
* TBD - `rxStatus.Status`
* TBD - `rxAuth.Auth`
* #1536 - refactor(Permission): FRMW-409-convert to Quark Service
* #1535 - refactor(quarks): FRMW-405 Page and FRMW-406 Paginate filters
* #1534 - refactor(quarks): FRMW-407 rxPageTracking FRMW-408 rxPaginateUtils.
* #1533 - refactor(quarks): FRMW-386 rxAppRoutes.rxAppRoutes FRMW-387 urlUtils
* #1530 - refactor(encoreRoutes): FRMW-381-convert to Quark Service.
* #1529 - refactor(rxStatusTags): FRMW-385-convert to Quark Service.
* #1528 - refactor(rxVisibilityPathParams): FRMW-384-convert to Quark Service.
* #1526 - refactor(UnauthorizedInterceptor): FRMW-377 convert to Quark Service.
* #1524 - refactor(TokenInterceptor): FRMW-376-convert to Quark Service.

File: src/components/rxEnvironment/rxEnvironment.module.js [hunk 1]
```diff
@@ -62,9 +62,6 @@
  *
  * ## Directives
  * * {@link rxEnvironment.directive:rxEnvironment rxEnvironment}
- *
- * ## Services
- * * {@link quarks.service:Environment Environment}
```
Comment (100stacks) [inline src/components/rxEnvironment/rxEnvironment.module.js]: Removing API documentation link to `Quarks` from `rxEnvironment.module.js`.

**Comment by 100stacks** [comment]
## Full Quarks TOC

screen shot 2016-02-16 at 12 51 53 pm

**Comment by CITguy** [comment]
LGTM

**Pull request merged**